src/main/java/mate/academy/spring/boot/service/impl/BookServiceImpl.java

```diff
@@ -41,7 +44,20 @@ public BookDto findById(Long id) {
 
     @Override
     public List<BookDto> getAllByTitle(String title) {
-        return bookRepository.findAllByTitle(title).stream()
+        return bookRepository.findAllByTitleContainsIgnoreCase(title).stream()
```

Comment:
Just a thoughts, if you implement searchBooks() and use TitleSpecProvider, it could be redundant to use findAllByTitleContainsIgnoreCase(). I suggest to change it to getAllBooks() without particular filter property. In addition, task description required exactly findAll method without any criteria.